aspnetcore/signalr/configuration.md

```diff
@@ -349,6 +351,7 @@ Additional options can be configured in the `WithUrl` (`withUrl` in JavaScript)
 | `logMessageContent` | `null` | Set to `true` to log the bytes/chars of messages sent and received by the client. |
 | `skipNegotiation` | `false` | Set this to `true` to skip the negotiation step. **Only supported when the WebSockets transport is the only enabled transport**. This setting can't be enabled when using the Azure SignalR Service. |
 | `withCredentials` | `true` | Specifies whether credentials will be sent with the CORS request. Azure App Service uses cookies for sticky sessions and needs this option enabled to work correctly. For more info on CORS with SignalR, see <xref:signalr/security#cross-origin-resource-sharing>. |
+| `timeout` | `100,000` | Timeout in milliseconds to apply to HTTP requests. This does not apply to Long Polling poll requests, EventSource, or WebSockets. |
```

Comment:
```suggestion
| `timeout` | `100,000` | Timeout in milliseconds to apply to HTTP requests. This doesn't apply to Long Polling poll requests, EventSource, or WebSockets. |
```

Reply:
Do we prefer contractions for better MT? So instead of translating "does" and "not" it translates "does not"?

Reply:
I don't think that rule comes from the [MT guide](https://review.docs.microsoft.com/en-us/help/contribute/localization-mt-guidance?branch=master).  I think that comes from one of our other style guides. @tdykstra  might know where that rule comes from.

Reply:
https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=36389

> Use common contractions, such as it’s, you’re, that's, and don’t, to create a friendly, informal tone.